scripts/metrics/common/config.py

```diff
@@ -0,0 +1,79 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Config file utilities for metrics scripts
+"""
+
+import os
+import sys
+
+import yaml
+
+from common.exception import MetricsExn
+
+config = {}
+
+
+class ConfigExn(MetricsExn):
+    pass
+
+
+def load_config(local_path, override=None):
+    global config
+
+    if override is None:
+        override = {}
```

Comment:
Instead of doing this, why not just set ```{}``` as the default value if it is not supplied and remove this check. Assigning argument variables is not a good idea in any case

Reply:
It's best practice to use None as the default arg because the default is only evaluated once at function definition. Multiple uses of the function may result in incorrect behavior.

Reply:
Alternatively then, you can do 
```
override = {} if override is None else override
```